### DISCUSSION TITLE
Links to discussions in other orgs does not always do the right thing™️

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#27403] [Discussions]
# Steps

1. Visit https://github.com/orgs/community/discussions/15740
    <img width="946" alt="image" src="https://user-images.githubusercontent.com/2119212/181399169-e39db5b8-90c1-42ce-8a09-6aebc83ad4c8.png">

2. Click the link `https://github.com/jsoref/discussions-1/discussions/1`

# Actual Results

`https://github.com/orgs/community/discussions/3` -- https://github.com/orgs/community/discussions/3

<img width="953" alt="image" src="https://user-images.githubusercontent.com/2119212/181399217-9611430c-0131-4d7e-922e-3b1b810bfb1b.png">

# Expected Results

`https://github.com/jsoref/discussions-1/discussions/3` -- https://github.com/jsoref/discussions-1/discussions/3

<img width="921" alt="image" src="https://user-images.githubusercontent.com/2119212/181399333-994c7228-15b0-4643-8c84-3c4b9ddc9c96.png">

---

> **Note**
> Here's what I currently see for `Expected Results`:
> <img width="894" alt="image" src="https://user-images.githubusercontent.com/2119212/181399608-0a509d35-f78d-4958-8349-fa112830660d.png">

**Comment by setharnold** [comment]
I saw the same thing with Firefox 102 on Ubuntu, logged in to my account.

**Comment by bewuethr** [comment]
Maybe related: the links to discussions in my org from the GitHub notifications tab are broken; instead of taking me to the repo with the linked discussion, they all go to the repo that is configured to host the org-level discussions.
